index.js

```diff
@@ -1,7 +1,90 @@
-// Iteration 1: Names and Input
-
-
-// Iteration 2: Conditionals
-
-
-// Iteration 3: Loops
+//1.ITERATION//
+let hacker1 = "Alberto";
+console.log("The driver's name is", hacker1);
+
+let hacker2 = "Juan";
+console.log("the navigator's name is", hacker2);
+
+//2.CONDITIONALS//
+let longitudHacker1 = hacker1.length;
+let longitudHacker2 = hacker2.length;
+
+if (longitudHacker1 > longitudHacker2) {
+  console.log("The driver has the longest name, it has " + longitudHacker1 + " characters.");

```

Comment:
Esto esta bien! pero te pongo otra forma de hacerlo! :D

```suggestion
  console.log(`The driver has the longest name, it has ${longitudHacker1} characters.`);
```